scripts/datadotparl/update-members

```diff
@@ -71,6 +71,13 @@ class Position(object):
     self.position = position
     self.list = positions
 
+  def chair_check(self, chair_date, periods, to_date):
+    if chair_date[1] < to_date:
+      periods.append({'position': 'Chair', 'fromdate': chair_date[0], 'todate': chair_date[1]})
+      periods.append({'position': '', 'fromdate': chair_date[1], 'todate': to_date})
+    else:
+      periods.append({'position': 'Chair', 'fromdate': chair_date[0], 'todate': to_date})
+
```

Comment:
I take it from this that we are assuming that a Chair is in place for a single stretch and any gaps are due to the dissolution of Parliament for elections?